templates/conference/debugpanel/index.html

```diff
@@ -15,5 +15,11 @@
       <tr><td>{{ key }}</td><td>{{ val }}</td></tr>
       {% endfor %}
     </table>
+
+    <h4>Additional links</h4>
+    <ul>
+      <li><a href='{% url "debug_panel_invoice_placeholders" %}'>Invoices placeholders (VAT ID 2018)</a></li>
```

Comment:
maybe rename the name of the view, `debug_panel_invoice_placeholders` is not specific to 2018.

Reply:
placeholders in itself are not specific to 2018, we may for example use exactly the same setup next year. (hopefully not, but it would work exactly the same, and None of the placeholder features are strictly tied to 2018)